frontend/test/app/components/ExperimentDisableDialog-test.js

```diff
@@ -27,8 +28,8 @@ describe('app/components/ExperimentDisableDialog', () => {
 
   it('should render a modal container', () => {
     expect(subject.find('.modal-container')).to.have.property('length', 1);
-    expect(subject.find('.modal-header').props()['data-l10n-args'])
-      .to.equal(JSON.stringify({ title: experiment.title }));
+    expect(findLocalizedById(subject, 'feedbackUninstallTitle').props()['$title'])
```

Comment:
Yes, we could do that, but for some reason looking for localization ids just seems like the right way to do it to me. It's already there, it's important because it determines the text the user sees there, makes sense to use it.